roseau/load_flow/models/core.py

```diff
@@ -112,8 +124,8 @@ def _switch_class(cls) -> type["Switch"]:
     def __init__(
         self,
         id: Any,
-        n1: int,
-        n2: int,
+        phases1: Phases,
+        phases2: Phases,
```

Comment:
Maybe we can make them optional and default to `bus1.phases` and `bus2.phases` ?